server/src/main/resources/schema/jboss-as-config_1_5.xsd

```diff
@@ -3353,7 +3353,7 @@
         <xs:complexContent>
             <xs:extension base="base-scoped-roleType">
                 <xs:choice>
-                    <xs:element name="host" type="namedType" minOccurs="1" maxOccurs="unbounded">
```

Comment:
We can not change any schema that has already been released.
That means that the only change we can make is in the new wildfly-config_13_0.xsd that I added last week that will be released with WildFly Core 13 (for WildFly 20)

Reply:
pushed a new change to only update that value in new wildfly-config_13_0.xsd